tools/clang/test/DXC/embed_dbg.test

```diff
@@ -2,7 +2,8 @@
 // RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Qembed_debug /Fo %t.embedpdb
 // RUN: %dxc -dumpbin %t.embedpdb | FileCheck  %s --check-prefix=EMBED_PDB
 
-// RUN: ls -1 %T | not grep "\\.pdb$"
+// RUN: ls -1 %T | FileCheck %s --check-prefix=PDB_NAME
```

Comment:
How is there any chance of it randomly writing a pdb into this particular folder?  Originally, the check was for the working directory, not the folder containing the test, wasn't it?